app/components/concept_details_component/concept_details_component.html.haml

```diff
@@ -6,6 +6,7 @@
     - if @bottom_keys.present?
       = render DropdownContainerComponent.new(title: 'Raw data', id: "accordion-#{@id}") do
         - top_set, leftover_set, bottom_set = filter_properties(@top_keys, @bottom_keys, @exclude_keys, prefix_properties(@concept_properties))
+        - leftover_set = convert_dates(leftover_set)
```

Comment:
if you use `convert_dates` only here, don't put the function in the `application_helper.rb`, just put the function `concept_details_component.rb`

Reply:
done